docs/mail.md

```diff
@@ -0,0 +1,27 @@
+# Mail
+
+Any community needs to send emails to allow for email verification, password resets, notifications, and other communication to users. Configuring your forum to send emails should be one of your first steps as an admin: an incorrect configuration will cause errors when users try to register.
+
+## Available Drivers
+
+Flarum provides several drivers by default, they are listed and explained below. Developers can also add [custom mail drivers through extensions](extend/mail.md).
+
+### SMTP
+
+This is probably the most commonly used email driver, and works pretty much exactly like you'd expect. Please note that the encryption field expects either `ssl` or `tls`.
```

Comment:
I would maybe say something like "configure your host, port, username and password for SMTP" instead of "what you'd expect".